applications/StructuralMechanicsApplication/custom_utilities/tangent_operator_calculator_utility.h

```diff
@@ -355,20 +358,20 @@ class TangentOperatorCalculatorUtility
         double& rMaxValue
         )
     {
-        const std::size_t size1 = rMatrixValues.size1();
-        const std::size_t size2 = rMatrixValues.size2();
+        const SizeType size1 = rMatrixValues.size1();
+        const SizeType size2 = rMatrixValues.size2();
         std::vector<double> non_zero_values;
 
-        for (std::size_t i = 0; i < size1; ++i) {
-            for (std::size_t j = 0; j < size2; ++j) {
+        for (IndexType i = 0; i < size1; ++i) {
+            for (IndexType j = 0; j < size2; ++j) {
                 if (std::abs(rMatrixValues(i, j)) > tolerance)
                     non_zero_values.push_back(std::abs(rMatrixValues(i, j)));
```

Comment:
u would REALLy not use a push back for this...  you are allocating memory for something that does not need allocation!

also, why is it a problem if the F is full of zeros?

Reply:
I think there is a check, I will check